calico/maintenance/ebpf/install.md

```diff
@@ -282,7 +281,7 @@ When the main install guide tells you to apply the `custom-resources.yaml`, typi
 the URL of the file directly, you should instead download the file, so that you can edit it:
 
 ```bash
-curl -o custom-resources.yaml <url of the file from the main install guide>
+curl -o custom-resources.yaml {{site.data.versions.first.manifests_url}}/manifests/custom-resources.yaml
```

Comment:
Not sure this is right.  Don't we have multiple flavours of custom-resources.yaml for different kinds of clusters?  (Or at least, it needs to be patched in some cases.)

Reply:
I think there is only a general  custom-resources. YAML file and for different cases we instruct people to modify it, for example here after users curl the file in this particular step they get an explanation on what needs to be modified.